util/fusermount.c

```diff
@@ -1374,9 +1375,11 @@ int main(int argc, char *argv[])
 	int cfd;
 	const char *opts = "";
 	const char *type = NULL;
+	int setup_auto_unmount_only = 0;
 
 	static const struct option long_opts[] = {
 		{"unmount", no_argument, NULL, 'u'},
+		{"setup-auto-unmount-only", no_argument, NULL, 'U'},
```

Comment:
I'm more than open to other suggestions for how the option (both long and short versions) should be named.